engine/classes/Elgg/Database/Plugins.php

```diff
@@ -410,11 +402,10 @@ function find($status = 'active') {
 			'type' => 'object',
 			'subtype' => 'plugin',
 			'limit' => ELGG_ENTITIES_NO_VALUE,
-			'selects' => array('plugin_oe.*', 'ps.value'),
```

Comment:
ps.value is still intact only attributes that were converted are title/description from the object... those were already never stored in private settings... don't think i need to do things with this... but will test later